src/soter/openssl/soter_rsa_key_3.c

```diff
@@ -278,10 +262,12 @@ soter_status_t soter_engine_specific_to_rsa_priv_key(const soter_engine_specific
     if (res != SOTER_SUCCESS) {
         /* Zero output memory to avoid leaking private key information */
         soter_wipe(key, output_length);
+        /* We did not use whole buffer, only `rsa_mod_size` bytes of it */
+        soter_wipe(bigint_buf, rsa_mod_size);
     }
 
 err:
-    BN_clear_free(tmp);
+    BN_free(rsa_e);
```

Comment:
Considering that these changes were primarily for buffer zeroizing, I don't see any zeroization :)

Reply:
Now this bigint only stores one public value, no need to `clear` it. Actual clearing was moved to `soter_wipe(bigint_buf, rsa_mod_size)`

Reply:
oh, sorry, didn't noticed `soter_wipe` call